nixos/modules/services/matrix/hebbot.nix

```
@@ -20,6 +20,14 @@ in
     meta.maintainers = [ lib.maintainers.raitobezarius ];
     options.services.hebbot = {
       enable = mkEnableOption "hebbot";
+      package = mkOption {
+        type = types.package;
+        default = pkgs.hebbot;
+        defaultText = lib.literalExpression "pkgs.hebbot";
+        description = ''
+          hebbot package to use.
+        '';
+      };
```

Can be simplified with ´lib.mkPackageOption´.

```suggestion
      package = lib.mkPackageOption pkgs "hebbot";
```